moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua

```diff
@@ -83,18 +83,14 @@ if read_config("enable_metrics") then
     secm = require "heka.secmetrics".new()
 end
 
-
 function genpayload()
     local msg = decode_message(read_message("raw"))
     local p = {}
+    table.sort(msg.Fields, function(a,b) return a.name < b.name end)
     for _, kvpair in ipairs(msg.Fields) do
-        if #kvpair.value == 1 then
-            p[kvpair.name] = kvpair.value[1]
-        else
-            p[kvpair.name] = kvpair.value
-        end
+        table.insert(p, string.format("%s: %s", kvpair.name, tostring(kvpair.value[1])))
```

Comment:
Performance tip: direct index insertion is much faster (in this case most of the cost is in the string.format so it is not a huge win here)
``` lua
for i, kvpair in ipairs(msg.Fields) do
  p[i] = string.format("%s: %s", kvpair.name, tostring(kvpair.value[1]))
end
```